docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,163 @@
+
+### Data Import
+* Handle the data import process carefully, breaking it down into specific tasks such as configuring databases and adjusting the data import setup to work with the new store.
+* Ensure any existing databases, such as the one from another country in one case, are correctly renamed or adjusted to fit the new multi-DB structure.
+* Anticipate and plan for potential updates that may arise after end2end testing of the project data migration.
+
+### Code Buckets
+* If used, investigate and adjust code bucket configurations as necessary. The technical steps required for these adjustments should be documented thoroughly, ensuring that code-bucket related functionalities are not disrupted by the addition of a new store.
+
+### Cloud environment and monitoring
+* Think of and adjust monitoring tools and APM (such as NewRelic, CloudWatch) to accommodate the new store. Check that all alerts and metrics are correctly configured to monitor the health and performance of the new store alongside existing ones.
+* Think of adjusting AWS Services such as S3, introducing buckets for the new store(s).
+
+### Front-end Considerations
+* Consider any other activities related to the above epics that might impact the deployment. For instance, front-end separation might be a significant task, requiring layout adjustments between different stores and possible adjustments on the API side.
+
+## Step-by-Step Procedure to release a new store(s)
+
+Follow [this guideline](/docs/scos/dev/technical-enhancement-integration-guides/integrate-multi-database-logic.html#define-databases) as a generic technical guideline, which is a requirement from the cloud team, to successfully deploy your new store in a multi-db setup.
+
+### Local Setup
+#### New store configuration
+
+* Define a new database and the store in the deploy file, following [that guide](/docs/ca/dev/multi-store-setups/add-and-remove-databases-of-stores.html#remove-the-configuration-of-the-database). As a result you should have:
+  * new database in regions.<region_name>.services.databases 
+  * new store in regions.<region_name>.stores 
+  * new domains in groups.<region_name>.applications  
+* Adjust stores.php with the configurations, relevant for your new store, following generic technical guideline. 
+* Prepare data import configurations and data files, specific to the new store. 
+* Adjust the local environment setup as needed, including configurations and environment variables. Examples: frontend router configuration, code bucket configuration, creating new backoffice users. 
+* Document all the steps you have done, to make sure they are repeatable in the future.
+
+#### Running Initial Setup
+
+* Boot the new configurations using the appropriate setup tool. 
+* Execute commands to initialize the database and services for the new store. 
```

Comment:
which exactly? you never asked to prepare them.

Reply:
Thanks, Changed sentences. THere's nothing special but running boot & up